build-tools/xa-prep-tasks/xa-prep-tasks.targets

```diff
@@ -67,6 +69,18 @@
         DestinationFiles="$(_BundlePath)"
     />
   </Target>
+  <Target Name="_DownloadNuGet"
+	Inputs=""
+	Outputs="$(_NuGetPath)\NuGet.exe">
+    <MakeDir
+        Directories="$(_NuGetPath)"
+    />
+    <DownloadUri
+        ContinueOnError="True"
```

Comment:
Do we actually want to continue on error? If the URL can't be downloaded, will tests actually execute? Will things "work"?